csharp/Platform.IO/ConsoleHelpers.cs

```diff
@@ -38,9 +82,29 @@ public static string GetOrReadArgument(int index, string readMessage, params str
             }
         }
 
+        /// <summary>
+        /// <para>Output a input string to the console. The method is only executed if the application has been compiled with the DEBUG directive.</para>
+        /// <para>Выводит переданную строку в консоль. Метод выполняется только в том случае, если приложение было скомпилировано с директивой DEBUG.</para>
+        /// </summary>
+        /// <param name="string">
+        /// <para>The string, that you want to output to the console.</para>
+        /// <para>Строка, которую нужно вывести в консоль.</para>
+        /// </param>
         [Conditional("DEBUG")]
         public static void Debug(string @string) => Console.WriteLine(@string);
 
+        /// <summary>
+        ///	<para>Output composit formatted text in the console.</para>
+        /// <para>Выводит строку составного формата в консоли.</para>
```

Comment:
[The same](https://docs.microsoft.com/ru-ru/dotnet/api/system.console.writeline?view=net-5.0#System_Console_WriteLine_System_String_System_Object___) with the Russian version.